src/handlers/notification/index.js

```diff
@@ -98,40 +103,40 @@ const processMessage = async (msg) => {
     Logger.info('Notification::processMessage action: ' + action)
     Logger.info('Notification::processMessage status: ' + status)
     if (action === 'prepare' && status === 'success') {
-      let callbackURL = Mustache.render(Config.DFSP_URLS[to].transfers.post, { transferId: id })
+      let callbackURL = Mustache.render(await Helper.getEndpoint(server, to, FSPIOP_CALLBACK_URL_TRANSFER_POST), { transferId: id })
```

Comment:
I do not think that we should do the Mustache.render here.

I think we should have a Facade to call the DAO and then render the URI.